restdocs-api-spec-openapi-generator/src/main/kotlin/com/epages/restdocs/apispec/openapi2/OpenApi20Generator.kt

```diff
@@ -267,16 +267,18 @@ object OpenApi20Generator {
             produces = modelsWithSamePathAndMethod.map { it.response.contentType }.distinct().filterNotNull().nullIfEmpty()
             parameters =
                     extractPathParameters(
-                        firstModelForPathAndMethod
+                        modelsWithSamePathAndMethod
```

Comment:
I think for the path parameters this changes is not needed. All items in `modelsWithSamePathAndMethod`  have the same path. This is asserted before. So it is save to just consider the first item in case of the path parameters